### PR TITLE
implement subscribe section

### PR DESCRIPTION
**Pull request by mailtime-wing**
please review, thanks

### CODE REVIEW

File: src/components/GetStarted/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const GetStarted = () => (
+  <div css={container}>
+    <div css={titleContainer}>
+      <PurpleRightArrow />
+      <p css={theme => [fontStyle(theme), title]}>
+        <FormattedMessage
+          id="readyToGetStarted"
+          defaultMessage="Ready to <p>Get STARTED ?</p>"
+          values={{
+            p: str => <p css={theme => [fontStyle(theme), subTitle]}>{str}</p>,
```
Comment (chochihim) [inline src/components/GetStarted/index.js]: `<p />` should not be under `<p />`. Please use another suitable element

File: src/pages/index.js [hunk 2]
```diff
@@ -21,6 +23,12 @@ const lastSection = css`
   margin-bottom: 30rem;
 `
 
+const subscribeSection = css`
+  margin: 0 auto;
+  width: 102.4rem;
+  padding: 0 1.2rem;
+`
```
Comment (chochihim) [inline src/pages/index.js]: Add following to match design:
```suggestion
box-sizing: border-box;
`
```

File: src/components/Subscribe/index.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+          values={{
+            span: str => <span css={subscribe}>{str}</span>,
+          }}
+        />
+      </p>
+    </div>
+    <div css={contentContainer}>
+      <div css={contentLeftContainer}>
+        <div css={emailAddressContainer}>
+          <p css={emailAddress}>
```
Comment (chochihim) [inline src/components/Subscribe/index.js]: This should be a `<input />` element

File: src/components/Subscribe/index.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            <FormattedMessage
+              id="emailAddress"
+              defaultMessage="Email Address"
+            />
+          </p>
+          <GreenRightArrow />
```
Comment (chochihim) [inline src/components/Subscribe/index.js]: I find that this arrow is different from others (the dots needs to be larger for this one). Use another image for it.

File: src/components/GetStarted/index.js (hunk 1 above)
Comment (mailtime-wing) [reply src/components/GetStarted/index.js]: fixed.

File: src/pages/index.js (hunk 2 above)
Comment (mailtime-wing) [reply src/pages/index.js]: added.

File: src/components/Subscribe/index.js (hunk 3 above)
Comment (mailtime-wing) [reply src/components/Subscribe/index.js]: fixed.

File: src/components/Subscribe/index.js (hunk 4 above)
Comment (mailtime-wing) [reply src/components/Subscribe/index.js]: changed.

**Pull request merged**